pulsar-transaction/coordinator/src/main/java/org/apache/pulsar/transaction/coordinator/impl/MLTransactionMetadataStore.java

```diff
@@ -247,8 +247,11 @@ public CompletableFuture<TxnID> newTransaction(long timeOut) {
                         .setMetadataOp(TransactionMetadataEntry.TransactionMetadataOp.NEW)
                         .setLastModificationTime(currentTimeMillis)
                         .setMaxLocalTxnId(sequenceIdGenerator.getCurrentSequenceId());
+                String id = UUID.randomUUID().toString();
+                System.out.println(id + " start");
                 transactionLog.append(transactionMetadataEntry)
                         .whenComplete((position, throwable) -> {
+                            System.out.println(id + " end");
```

Comment:
```suggestion
                         if (log.isDebugEnabled()) {
                                log.debug("Transaction coordinator [{}] complete to open transaction [{}]",
                                        txnID.getMostSigBits(), txnID.getLeastSigBits());
                            }
```

Reply:
This is the debug problem log, which has been deleted.

Thanks